app/templates/main/index.html

```diff
@@ -12,6 +18,5 @@
         called <a href="http://semantic-ui.com">Semantic UI</a>. Much of this code is appropriated from the examples
         in Miguel Grinberg's book, <a href="http://flaskbook.com"><em>Flask Web Development</em></a>.</p>
 
-        <p>{{ lipsum(6) }}</p>
```

Comment:
I think I was just checking how it works. I don't think it will matter since we will be removing it later on anyways.